packages/metro/src/node-haste/DependencyGraph.js

```diff
@@ -223,6 +223,19 @@ class DependencyGraph extends EventEmitter {
     this._haste.end();
   }
 
+  /** Given a search context, return a list of file paths matching the query. */
+  resolveContext(
```

Comment:
This implementation doesn't resolve the paths (as in "module resolution"), so let's reflect that in the name.

```suggestion
  matchFilesWithContext(
```